modules/ossm-threescale-webassembly-module-credentials-object.adoc

```diff
@@ -0,0 +1,34 @@
+// Module included in the following assembly:
+//
+// service_mesh/v2x/ossm-threescale-webassembly-module.adoc
+
+[id="ossm-threescale-webassembly-module-credentials-object_{context}"]
+= The 3scale WebAssembly module `credentials` object
+
+The `credentials` object is a component of the `service` object. `credentials` specifies which kind of credentials to be looked up and the steps to perform this action.
+
+All fields are optional, but you must specify at least one, `user_key` or `app_id`. The order in which you specify each credential is irrelevant because it is pre-established by the module. Only specify one instance of each credential.
+
+----
+credentials:
+  user_key: <array_of_lookup_queries>
+  app_id: <array_of_lookup_queries>
+  app_key: <array_of_lookup_queries>
+----
```

Comment:
same comments as above re stating the [source,x] for code blocks and adding extra content or `...` to show context / excerpt